lib/commands/help.js

```diff
@@ -92,8 +75,9 @@ module.exports = Command.extend({
         }.bind(this));
       }
 
+      var multipleCommands = [GenerateCommand.prototype.name].concat(GenerateCommand.prototype.aliases);
       if (multipleCommands.indexOf(rawArgs[0]) > -1) {
-        command = rawArgs.shift();
+        var command = rawArgs.shift();
         if (rawArgs.length > 0) {
```

Comment:
this `if` is redundant inside [this `if`](https://github.com/Turbo87/ember-cli/blob/json/lib/commands/help.js#L43).